docs/ide/package/installer-projects-overview.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Visual Studio Installer Projects | Microsoft Docs"
+ms.date: "12/03/2019"
+ms.topic: "conceptual"
+helpviewer_keywords:
+  - "installer projects"
+  - "installer projects, overview"
+author: ruconard
+ms.author: ruconard
+manager: ruconard
+ms.workload:
+  - "multiple"
+---
+# Visual Studio Installer Projects Extension
+
+Packaging applications as an MSI is often accomplished using the Visual Studio Installer Projects Extension.
+
+You can download the extension here:
+[Visual Studio Installer Projects](https://marketplace.visualstudio.com/items?itemName=VisualStudioClient.MicrosoftVisualStudio2017InstallerProjects)
+
+## Update for .NET Core
```

Comment:
There's more to say here, but it sounds like you want to address that in a follow up change.